packages/app/src/runner/utils.ts

```diff
@@ -23,8 +23,8 @@ export function getReporterElement () {
   return getElementById(REPORTER_ID)
 }
 
-export function empty (el: HTMLElement) {
-  while (el.lastChild) {
+export function empty (el: HTMLElement | null) {
```

Comment:
Same here - `empty` looks like it's designed to empty a `HTMLElement` - this change reflects we don't understand the different workflows and states our system can be in. Rather than proceeding with "try to empty", we should ensure we only call `empty` when we know we need to.

So, I think the check should be placed where this was prompted. Something like:

```js
if (!someElement) {
  // someElement doesn't exist for X reason, return
  return
}

empty(someElement)
```